src/parser/retail/modules/items/thewarwithin/trinkets/MereldarsToll.tsx

```diff
@@ -0,0 +1,93 @@
+import ITEMS from 'common/ITEMS/thewarwithin/trinkets';
+import SPELLS from 'common/SPELLS/thewarwithin/trinkets';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import SPELL_CATEGORY from 'parser/core/SPELL_CATEGORY';
+import Abilities from 'parser/core/modules/Abilities';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import BoringItemValueText from 'parser/ui/BoringItemValueText';
+import { formatPercentage, formatNumber } from 'common/format';
+import { DamageIcon, VersatilityIcon, InformationIcon } from 'interface/icons';
+import Combatants from 'parser/shared/modules/Combatants';
+import { calculateSecondaryStatDefault } from 'parser/core/stats';
+
+/**
+ * Based on the stats provided on wowhead.
+ *
+ * https://www.wowhead.com/item=219313/mereldars-toll
+ */
+const MERELDARS_TOLL_BASE_ILVL = 437;
+const MERELDARS_TOLL_BASE_GAIN = 658;
+
+export default class MereldarsToll extends Analyzer.withDependencies({
+  abilities: Abilities,
+  combatants: Combatants,
+}) {
+  protected damage: number = 0;
+  protected externalUptimeTotal: number = 0;
```

Comment:
`this.externalUptimeTotal` can most likely be omitted, since it isn't used anywhere else.
```suggestion
```